src/pages/workspaces/workspace/notebooks/AnalysisLauncher.js

```diff
@@ -502,19 +501,20 @@ const AnalysisEditorFrame = ({
   useOnMount(() => {
     const cloudStorageDirectory = `gs://${bucketName}/notebooks`
 
-    const pattern = Utils.switchCase(toolLabel,
-      [tools.RStudio.label, () => '.*\\.Rmd'],
-      [tools.Jupyter.label, () => '.*\\.ipynb']
+    const patterns = Utils.switchCase(toolLabel,
+      [tools.RStudio.label, () => ['.*\\.Rmd', '.*\\.R']],
```

Comment:
Since the `setupWelder` function in AppLauncher.js also uses both of these patterns to make `Ajax().Runtimes.fileSyncing(...).setStorageLinks(...)` requests, is it worth moving these to a constant in notebook-utils.js?

Reply:
Great suggestion. I just added that change in my last push